examples/seismic/tutorials/07_DRP_schemes.ipynb

```diff
@@ -396,18 +396,14 @@
     "        x, z = dimensions\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #27.                   subdomain=ur)
</pre><p>i.e. to be created here....</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/2245/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>